### ISSUE TITLE
TTA Method in DeeplabV3+ | Evaluation 

### ISSUE THREAD

**Issue by chris3006** [nyoki-mtl/pytorch-segmentation#13]
What is the intension of the TTA method? Does it deliver better results?

I want to use this network as an input to another network and im not sure, whether I should use the model.tta method. 

```
with torch.no_grad():
    for batched in valid_loader:
        images, labels = batched
        images_np = images.numpy().transpose(0, 2, 3, 1)
        labels_np = labels.numpy()

        images, labels = images.to(device), labels.to(device)
        preds = model.tta(images, net_type='deeplab')
        preds = preds.argmax(dim=1)
        preds_np = preds.detach().cpu().numpy()

        images_list.append(images_np)
        labels_list.append(labels_np)
        preds_list.append(preds_np)

        if len(images_list) == 4:
            break
```

**Comment by nyoki-mtl** [comment]
TTA stands for Test Time Augmentation.
If you want to improve accuracy, please use TTA.
If response speed is important, please use pred_resize method.
https://github.com/nyoki-mtl/pytorch-segmentation/blob/7a30cabb1e169f819329a5e3bd9d39f21d41c1c1/src/eval_cityscapes.py#L32
